docs/data/sql_functions.yml

```diff
@@ -561,7 +561,10 @@ conditional:
 conversion:
   - sql: CAST(value AS type)
     table: ANY.cast(TYPE)
-    description: Returns a new value being cast to type type. E.g., CAST('42' AS INT) returns 42; CAST(NULL AS VARCHAR) returns NULL of type VARCHAR.
+    description: Returns a new value being cast to type type. A CAST error throws an exception and fails the job. If you're performing a cast operation that may fail, like INT to STRING, you should rather use TRY_CAST, in order to handle errors. E.g., CAST('42' AS INT) returns 42; CAST(NULL AS VARCHAR) returns NULL of type VARCHAR; TRY_CAST('non-number' AS INT) throws an exception and fails the job.
```

Comment:
remove `TRY_`

Reply:
This has been fixed already in one of the last commits of this PR